apps/api/src/api.controller.ts

```diff
@@ -28,8 +40,23 @@ export class ApiController {
     description: 'Asset files',
     type: FilesUploadDto,
   })
-  // eslint-disable-next-line no-undef
-  async assetUpload(@UploadedFiles() files: Array<Express.Multer.File>): Promise<UploadResponseDto> {
+  async assetUpload(
+    @UploadedFiles(
+      new ParseFilePipeBuilder()
+        .addFileTypeValidator({
+          fileType: DSNP_VALID_MIME_TYPES,
+        })
+        .addMaxSizeValidator({
+          // this is in bytes (2 GB)
+          maxSize: 2 * 1000 * 1000 * 1000,
```

Comment:
Should be coming from config. I can either handle in this PR or in a future PR.

Reply:
Can we add a story or some bullet point to existing story to keep a track

Reply:
created a new ticket https://github.com/AmplicaLabs/content-publishing-service/issues/20